js/base.js

```diff
@@ -45,6 +45,7 @@ export class API extends mix(BaseAPI).with(PickupAPI, ShipmentAPI, TrackingAPI)
      * made by the API client
      * @returns {Object|String|Blob} The parsed result value for the provided
      * response object taking into account the content type of it.
+     * @see https://developer.dhl.com/api-reference/dhl-express-mydhl-api#reference-docs-section/operations-pickup-exp-api-pickups

```

Comment:
Notice that this does not make sense. This function of ours is not documented in that link (DHL API documentation)